src/pywifes/wifes_wsol.py

```diff
@@ -299,43 +309,22 @@ def weighted_loggauss_arc_fit(subbed_arc_data,
             # MJI: with the following test, the code ran faster, but even just the Pool(cpu)
             # command slowed things down. 
             #jobs2 = [jobs[:len(jobs)//4], jobs[len(jobs)//4:2*len(jobs)//4],jobs[2*len(jobs)//4:3*len(jobs)//4],jobs[3*len(jobs)//4:]]
-            #results = mypool.imap_unordered(utils.lsq_gauss_line,jobs2)
+            #results = mypool.imap_unordered(utils._lsq_gauss_line,jobs2)

```

Comment:
Same for this commented line. Just remove it.

Reply:
Yup, agreed. I've got more refactors in the next PR, but certainly anything I'm hitting in this MR should be cleaned up now.